pyproject.toml

```diff
@@ -143,23 +143,32 @@ parentdir_prefix = "pandas-"
 setup = ['--vsenv'] # For Windows
 
 [tool.cibuildwheel]
-skip = "cp36-* cp37-* pp37-* *-manylinux_i686 *_ppc64le *_s390x *-musllinux*"
+skip = "cp36-* cp37-* cp38-* pp* *_i686 *_ppc64le *_s390x *-musllinux_aarch64"
 build-verbosity = "3"
-environment = {LDFLAGS="-Wl,--strip-all" }
+environment = {LDFLAGS="-Wl,--strip-all"}
 test-requires = "hypothesis>=6.46.1 pytest>=7.0.0 pytest-xdist>=2.2.0 pytest-asyncio>=0.17"
-test-command = "python {project}/ci/test_wheels.py"
+test-command = """
+  PANDAS_CI='1' python -c 'import pandas as pd; \
+  pd.test(extra_args=["-m not clipboard and not single_cpu and not slow and not network and not db", "-n 2"]); \
+  pd.test(extra_args=["-m not clipboard and single_cpu and not slow and not network and not db"]);' \
+  """
 
 [tool.cibuildwheel.macos]
 archs = "x86_64 arm64"
 test-skip = "*_arm64"
 
 [tool.cibuildwheel.windows]
-repair-wheel-command = "python ci/fix_wheels.py {wheel} {dest_dir}"
+before-build = "pip install delvewheel"
+repair-wheel-command = "delvewheel repair -w {dest_dir} {wheel}"
```

Comment:
> Have you checked that this puts the DLLs in the same place?

Sorry I'm not really sure how to check this. I just assumed since the tests passed on windowsservercore the DLLs were correctly available. What's bad about a "weird" folder structure?

Reply:
I went and downloaded the wheel (not on a Windows computer, don't have access to that anymore, unfortunately).

It looks like they put DLLS into a separate folder from the window extension module, and then mess around with the DLL search path in __init__.py to make it work correctly (what I meant by weird, not necessarily bad, though).

See screenshots
![image](https://github.com/pandas-dev/pandas/assets/47963215/e542bdae-54ad-414a-8f24-c369141a193f)
![image](https://github.com/pandas-dev/pandas/assets/47963215/e8763f1b-7527-4d29-b924-c92a046c953c)

Can you comment out the wheel repair step on Windows and make sure the wheel build fails with a missing DLL import error just as a sanity check that that works?

Reply:
Just got back to this. I commented out the wheel repair step in [53c03b7](https://github.com/pandas-dev/pandas/pull/53087/commits/53c03b7eb9bc402d09cde10ec15b24b32903a17c) and it raises

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python\lib\site-packages\pandas\__init__.py", line 46, in <module>
    from pandas.core.api import (
  File "C:\Python\lib\site-packages\pandas\core\api.py", line 47, in <module>
    from pandas.core.groupby import (
  File "C:\Python\lib\site-packages\pandas\core\groupby\__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import (
  File "C:\Python\lib\site-packages\pandas\core\groupby\generic.py", line 69, in <module>
    from pandas.core.frame import DataFrame
  File "C:\Python\lib\site-packages\pandas\core\frame.py", line 141, in <module>
    from pandas.core.generic import (
  File "C:\Python\lib\site-packages\pandas\core\generic.py", line 181, in <module>
    from pandas.core.window import (
  File "C:\Python\lib\site-packages\pandas\core\window\__init__.py", line 1, in <module>
    from pandas.core.window.ewm import (
  File "C:\Python\lib\site-packages\pandas\core\window\ewm.py", line 11, in <module>
    import pandas._libs.window.aggregations as window_aggregations
ImportError: DLL load failed while importing aggregations: The specified module could not be found.
```